### DISCUSSION TITLE
Merge items from Collection view

### DISCUSSION THREAD

**Discussion by justrealmilk** [directus/directus#10458] [Feature Requests]
An additional action button would live beside edit, delete, etc., when multiple items are selected. 

"Merge" would allow the user to deduplicate and merge the selected into a single item. 

Clicking "merge" would open a view very similar to the multiple item edit view and would allow the user to select which values should take precedence. 

For relational fields, the user may select multiple parent items to merge the children of. 

Thoughts? 🦘

**Comment by rijkvanzanten** [comment]
Ohhh interesting one! Haven't heard a similar thing before. Curious to see how common of a use case this would be 🦘

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
